[refactor] Update generalization graphs part

MaxenceRobin opened the pull request:
This commit updates the part abouth generalization graphs, to match the new changes about them

MaxenceRobin committed:
Correcting mistkaes

Correcting mistakes in the French CaSys manual

MaxenceRobin committed:
Refactoring files about CasSys

Corrections and additions of files about CasSys

MaxenceRobin committed:
[refactor] Update generalization graphs part

This commit updates the part abouth generalization graphs, to match the new changes about them

martinec commented:
@MaxenceRobin Before merging the PR, it is necessary to solve the issues raised by Travis.

MaxenceRobin committed:
Translate generalization graph part into English

This commits adds the translation of the part about generalization graph into English as well as correcting some mistakes in both French and English manuals (see issue #18)

MaxenceRobin committed:
Resolve merge conflict

martinec commented:
@MaxenceRobin Your PR has conflicts that must be resolved see https://github.com/UnitexGramLab/unitex-doc-usermanual/pull/19/conflicts

MaxenceRobin committed:
Merge branch 'master' into refactor/GeneralizationGraphs

martinec commented:
@MaxenceRobin, the PR still have errors. See here for a hint: https://travis-ci.org/UnitexGramLab/unitex-doc-usermanual/builds/411845558#L7529
